packages/api-nest/src/repositories/highlight.repository.ts

```
@@ -0,0 +1,72 @@
+import { HighlightEntity } from '../highlight/entities/highlight.entity'
+import { IHighlightRepository } from './interfaces/highlight-repository.interface'
+
+/**
+ * TypeORM implementation of the IHighlightRepository interface
+ * Handles all data access operations for highlights
+ */
+@Injectable()
+export class HighlightRepository implements IHighlightRepository {
+  constructor(
+    @InjectRepository(HighlightEntity)
+    private readonly repository: Repository<HighlightEntity>,
```

ESLint

[error] Delete `,`

(prettier/prettier)